test/quark-core/QuarkWallet.t.sol

```diff
@@ -634,9 +724,6 @@ contract QuarkWalletTest is Test {
 
     /* ===== MultiQuarkOperation execution path tests ===== */
 
-    // TODO: test replayable txns
```

Comment:
I think we should keep this TODO since we don't yet have tests for multi quark operations with replayables in them.

Reply:
added back